unittests/tools/test_checkmarx_osa_parser.py

```diff
@@ -188,6 +188,6 @@ def test_checkmarx_osa_parse_file_with_no_libraryId_raises_ValueError(
             parser = CheckmarxOsaParser()
             parser.get_findings(my_file_handle, test)
             self.teardown(my_file_handle)
-            self.assertTrue(
-                "Invalid format: missing mandatory field libraryId:" in str(context.exception)
-            )
+        self.assertEqual(
+            "Invalid format: missing mandatory field libraryId", str(context.exception)
+        )
```

Comment:
Looks like this should be an `assertIn`?

```suggestion
        self.assertIn(
            "Invalid format: missing mandatory field libraryId", str(context.exception)
        )
```

Reply:
Both are valid. The question is, how strict are going to be?

Reply:
You're right, `assertEqual` is better so long as the comparison is correct. Just wanted to be double-check 😄